src/ncp/spinel.h

```diff
@@ -532,7 +532,12 @@ typedef enum
     /** Format: `A(T(ESLCcCbLL))`
      *  eui64, rloc16, age, inLqi ,aveRSS, mode, isChild. linkFrameCounter, mleCounter
      */
-    SPINEL_PROP_THREAD_NEIGHBOR_TABLE  = SPINEL_PROP_THREAD_EXT__BEGIN + 11,
+    SPINEL_PROP_THREAD_NEIGHBOR_TABLE   = SPINEL_PROP_THREAD_EXT__BEGIN + 11,
```

Comment:
I wonder why adding a white-space before `=` here?  
I think this makes it mis-aligned with the rest of the prop definitions.